examples/tools/bevymark.rs

```diff
@@ -0,0 +1,173 @@
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    mut materials: ResMut<Assets<ColorMaterial>>,
+    mut out_material_handle: ResMut<Option<Handle<ColorMaterial>>>,
+) {
+    *out_material_handle = Some(
+        materials.add(
+            asset_server
+                .load("assets/branding/icon.png")
+                .unwrap()
+                .into(),
+        ),
+    );
+
+    commands
+        .spawn(Camera2dComponents::default())
+        .spawn(UiCameraComponents::default())
+        .spawn(TextComponents {
+            text: Text {
+                font: asset_server.load("assets/fonts/FiraSans-Bold.ttf").unwrap(),
+                value: "Bird Count:".to_string(),
+                style: TextStyle {
+                    color: Color::rgb(0.0, 1.0, 0.0),
+                    font_size: 40.0,
+                },
+            },
+            style: Style {
+                position_type: PositionType::Absolute,
+                position: Rect {
+                    top: Val::Px(5.0),
+                    left: Val::Px(5.0),
+                    ..Default::default()
+                },
+                ..Default::default()
+            },
+            ..Default::default()
+        });
+}
+
+fn mouse_handler(
+    mut commands: Commands,
+    time: Res<Time>,
+    mouse_button_input: Res<Input<MouseButton>>,
+    window: Res<WindowDescriptor>,
+    material_handle: Res<Option<Handle<ColorMaterial>>>,
+    mut counter: ResMut<BevyCounter>,
+) {
+    if mouse_button_input.pressed(MouseButton::Left) {
+        let spawn_count = (BIRDS_PER_SECOND as f32 * time.delta_seconds) as u128;
+        let bird_x = (window.width as i32 / -2) as f32 + HALF_BIRD_SIZE;
+        let bird_y = (window.height / 2) as f32 - HALF_BIRD_SIZE;
+
+        for count in 0..spawn_count {
+            let bird_position = Vec3::new(bird_x, bird_y, (counter.count + count) as f32 * 0.00001);
+
+            commands
+                .spawn(SpriteComponents {
+                    material: material_handle.unwrap(),
+                    translation: Translation(bird_position),
+                    scale: Scale(BIRD_SCALE),
+                    draw: Draw {
+                        is_transparent: true,
+                        ..Default::default()
+                    },
+                    ..Default::default()
+                })
+                .with(Bird {
+                    velocity: Vec3::new(
+                        rand::random::<f32>() * MAX_VELOCITY - (MAX_VELOCITY * 0.5),
+                        0.,
+                        0.,
+                    ),
+                });
+        }
+
+        counter.count += spawn_count;
+    }
+}
+
+fn movement_system(time: Res<Time>, mut bird_query: Query<(&mut Bird, &mut Translation)>) {
+    for (mut bird, mut translation) in &mut bird_query.iter() {
+        translation.0 += bird.velocity * time.delta_seconds;
+
+        let new_y = bird.velocity.y() + GRAVITY;
+        bird.velocity.set_y(new_y);
```

Comment:
This makes gravity acceleration framerate dependent. We can correct this (and make it a bit terser) like this:

```rust
*bird.velocity.y_mut() += GRAVITY * time.delta_seconds;
```

This correction exposes something interesting: the scale is quite "off". You'll notice the birds appear to accelerate very slowly. This is because each pixel is currently a "meter", so the window is 800 "meters" tall.

We can fix this by just cranking up the gravity:
```rust
const GRAVITY: f32 = -9.8 * 100.0; // multiply by 100 to account for scale
```